damus/Util/Hashtags.swift

```diff
@@ -9,60 +9,70 @@ import Foundation
+    "bitcoin": CustomHashtag(imageName: "bitcoin", display: "Bitcoin", color: Color.orange, offset: -3.0),
+    "bitcoinwallet": CustomHashtag(imageName: "bitcoin", display: "Bitcoin", color: Color.orange, offset: -3.0),
+    "bitcoinadoption": CustomHashtag(imageName: "bitcoin", display: "BitcoinAdoption", color: Color.orange, offset: -3.0),
+    "bitcoinjungle": CustomHashtag(imageName: "bitcoin", display: "BitcoinJungle", color: Color.orange, offset: -3.0),
+    "btc": CustomHashtag(imageName: "bitcoin", display: "BTC", color: Color.orange, offset: -3.0),
+    "coffee": CustomHashtag(imageName: "coffee", display: "Coffee", color: DamusColors.brown, offset: -1.0),
+    "coffeechain": CustomHashtag(imageName: "coffee", display: "CoffeeChain", color: DamusColors.brown, offset: -1.0),
+    "nostr": CustomHashtag(imageName: "nostr", display: "Nostr", color: DamusColors.purple, offset: -2.0),
+    "nostrica": CustomHashtag(imageName: "nostr", display: "Nostrica", color: DamusColors.purple, offset: -2.0),
+    "onlyzap": CustomHashtag(imageName: "zap", display: "OnlyZap", color: DamusColors.yellow, offset: -4.0),
+    "onlyzaps": CustomHashtag(imageName: "zap", display: "OnlyZaps", color: DamusColors.yellow, offset: -4.0),
+    "plebchain": CustomHashtag(imageName: "plebchain", display: "PlebChain", color: DamusColors.deepPurple, offset: -3.0),
+    "zap": CustomHashtag(imageName: "zap", display: "Zap", color: DamusColors.yellow, offset: -4.0),
+    "zaps": CustomHashtag(imageName: "zap", display: "Zaps", color: DamusColors.yellow, offset: -4.0),
+    "zapathon": CustomHashtag(imageName: "zap", display: "Zapathon", color: DamusColors.yellow, offset: -4.0),
 ]
 
 func hashtag_str(_ htag: String) -> CompatibleText {
-    var attributedString = AttributedString(stringLiteral: "#\(htag)")
-    attributedString.link = URL(string: "damus:t:\(htag)")
-    
     let lowertag = htag.lowercased()
     
-    var text = Text(attributedString)
+    let displayText: String
+    var displayColor: Color?
+    var imageOffset: CGFloat?
+    var imageName: String?
+    
     if let custom_hashtag = custom_hashtags[lowertag] {
-        if let col = custom_hashtag.color {
-            attributedString.foregroundColor = col
-        }
-        
-        let name = custom_hashtag.name
+        displayText = custom_hashtag.display
+        displayColor = custom_hashtag.color
+        imageOffset = custom_hashtag.offset
+        imageName = custom_hashtag.imageName
+    } else {
+        displayText = htag
+        displayColor = DamusColors.purple
+        imageOffset = nil
+        imageName = nil
+    }
+    
+    var attributedString = AttributedString(stringLiteral: "#\(displayText)")
+    attributedString.link = URL(string: "damus:t:\(displayText)")
+    attributedString.foregroundColor = displayColor
+       
+    var text = Text(attributedString)
+    
+    if let imageName = imageName, let img = UIImage(named: "\(imageName)-hashtag") {
```

Comment:
```suggestion
    if let imageName, let img = UIImage(named: "\(imageName)-hashtag") {
```

Reply:
Applied, actually just wanted to do `if imageName != nil, let ...`